app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Using a class-level dictionary `Person.people` can lead to issues if multiple instances are created or if the function is called multiple times. Consider using instance-level storage or ensuring unique keys.